packages/chartjs-plugin/src/utils.ts

```diff
@@ -39,9 +39,28 @@ export const getDatasetColors = (color: OnyxColor | `quantitatives-${Quantitativ
  * Gets the current value of the given CSS variable.
  * @param CSS variable name, e.g. `--onyx-color-base-primary-500`
  */
-export const getCSSVariableValue = (variableName: string) => {
-  return getComputedStyle(document.body).getPropertyValue(variableName);
+
+export const getCSSVariableValue = (variableName: string): string => {
+  const isDark = getComputedStyle(document.body).colorScheme === "dark";
+  const computedValue = getComputedStyle(document.body).getPropertyValue(variableName).trim();
+  const lightDarkMatch = computedValue.match(/^light-dark\(([^,]+),\s*([^)]+)\)$/);
+
+  if (lightDarkMatch) {
+    const lightValue = lightDarkMatch[1].trim();
+    const darkValue = lightDarkMatch[2].trim();
+
+    if (isDark) {
+      return darkValue;
+    } else {
+      return lightValue;
+    }
+  } else {
+    return computedValue;
+  }
```

Comment:
```suggestion
  if (lightDarkMatch) {
    return lightDarkMatch[isDark ? 2 : 1].trim();
  } else {
    return computedValue;
  }
```

Can be simplified :)